src/mkdocstrings/plugin.py

```diff
@@ -103,6 +107,7 @@ class MkdocstringsPlugin(BasePlugin):
 
     def __init__(self) -> None:
         super(MkdocstringsPlugin, self).__init__()
+        # stop mypy from complaining about NoneType mismatches
```

Comment:
Shouldn't this be removed as well :smile_cat: ?

Reply:
Whoops!  Yup I'll fix that